Adds notes for 4.10.10 release and Telco RAN GA (#44654)
openshift/openshift-docs: release_notes/ocp-4-10-release-notes.adoc

==== Updating

To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.

[id="ocp-4-10-10"]
=== RHSA-2022:1357 - {product-title} 4.10.10 bug fix and security update

Issued: 2022-04-20

> {product-title} release 4.10.10 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1241[RHSA-2022:1357] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1357[RHBA-2022:1357] advisory.

Reviewer: First Errata link doesn't match text, and text is the same as the second link - wanna check these :)

Author: Whoops! I did this pre-coffee this morning. Updated!

Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:

link:https://access.redhat.com/solutions/6932751[{product-title} 4.10.10 container image list]

[id="ocp-4-10-10-bug-fixes"]
==== Bug fixes

> * Previously, cluster storage operator credentials request for Amazon Web Services (AWS) did not include KMS statements. Consequently, persistent volumes (PVs) failed to deploy due to the inability to provide a key. With this update, the default credentials request for AWS now allow mounting of encrypted volumes using customer managed keys from KMS. Administrators who create credentials requests in manual mode with Cloud Credential Operator (CCO), will need to apply those changes manually. Other administrators should not be impacted by this change. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072191[*BZ#2072191*])

Reviewer: Operator is always capitalized, idk if "cluster storage" is a proper name or not
```suggestion
* Previously, the cluster storage Operator credentials request for Amazon Web Services (AWS) did not include KMS statements. Consequently, persistent volumes (PVs) failed to deploy due to the inability to provide a key. With this update, the default credentials request for AWS now allows the mounting of encrypted volumes using customer-managed keys from KMS. Administrators who create credentials requests in manual mode with Cloud Credential Operator (CCO) must apply those changes manually. Other administrators should not be impacted by this change. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072191[*BZ#2072191*])
```

Author: I did a search and saw all lowercase, but capitalizing Operator is the right way to go. Updated!
